dace/sdfg.py

```diff
@@ -4433,8 +4433,15 @@ def replsym(symlist):
             if isinstance(propclass, properties.RangeProperty):
                 setattr(node, pname, replsym(propval))
             if isinstance(propclass, properties.CodeProperty):
-                for stmt in propval['code_or_block']:
-                    ASTFindReplace({name: new_name}).visit(stmt)
+                if isinstance(propval['code_or_block'], str):
+                    # TODO: C++ AST parsing for replacement?
+                    if name != str(new_name):
+                        warnings.warn(
```

Comment:
Are we sure we don't want to fail here? Won't this break in most cases?

Reply:
It will only break in cases where you have symbols (e.g. `a[i] = i`) in the tasklet itself. In most cases it doesn't break.

Reply:
Ok, sure